### ISSUE TITLE
Прилипание окон

### ISSUE THREAD

**Issue by syberexonline** [SBProX/SBProX-Dev#2056]
Прилипание 2х и более окон.
Уже во всех программах это есть по дефолту. Маст Хэв.

**Comment by serb81** [comment]
давно пора

**Comment by ElizarovSS** [comment]
Точно, на DELL мониторах такая фича по умолчанию есть в дровах, но почему то не подхватывает окна SB-шки они не активны для этого. А вообще неплохо было бы сделать.

**Comment by Gr00v35** [comment]
100% надо

**Comment by bitcony** [comment]
100%

**Comment by paskerplayer** [comment]
Нужна вещь
